### PR TITLE
Use webpack code splitting to lazy load fullCorvidTypes.json

### PR DESCRIPTION
**Pull request by dlvhdr**
image

### CODE REVIEW

File: src/dynamicTypes/typesLoader.ts [hunk 1]
```diff
@@ -18,9 +16,20 @@ type FullCorvidTypes = {
...
+}> => {
+  return import(
+    /* webpackChunkName: "fullCorvidTypesJSON" */ "../../dist/fullCorvidTypes.json"
+  );
+};
+
+export const getCollectionLibs = async (
+  key: LibCollections
+): Promise<Lib[]> => {
+  const fullCorvidTypesModule = await dynamicallyImportFullCorvidTypes();
```
Comment (nemni8) [inline src/dynamicTypes/typesLoader.ts:30]: tagid, this code doesn't mean that every time we are changing context we are importing the chunk again?
i think we should import it once

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -17,14 +17,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (dlvhdr) [reply .github/workflows/test.yml:20]: I just upgraded it.. I'm not sure, but as every software update, it probably has bug fixes :D

File: src/dynamicTypes/index.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
-  packages: require("./npmDependencies"),
-  elementsMap: require("./elementsMap"),
-  widget: require("./widget")
+  packagesDeclarations: require("./npmDependencies"),
```
[the remark this answers is not in the document]
Comment (dlvhdr) [reply src/dynamicTypes/index.js:2]: I tried, I'm not sure why it didn't succeed

File: src/dynamicTypes/typesLoader.ts (hunk 1 above)
Comment (dlvhdr) [reply src/dynamicTypes/typesLoader.ts:30]: No, webpack is smart to load it from the network only once.
Just checked with:

```
  console.time("corvid-types-import");
  const fullCorvidTypesModule = await dynamicallyImportFullCorvidTypes();
  console.timeEnd("corvid-types-import");
```

This results in similar times if I only call import once and store the value for caching or calling import every time.

**Pull request merged**